frontend/src/viewer/logic/score/league/czech.ts

```diff
@@ -0,0 +1,62 @@
+import { Measure } from '../measure';
+import { Score, scoreCircuits, scoreOpenDistance, scoreTriangles } from '../scorer';
+
+abstract class CZXCBase {
+  score(measure: Measure): Score[] {
+    return [
+      ...scoreOpenDistance(measure, 3, this.openDistanceMultiplier),
```

Comment:
Make `this.openDistanceMultiplier` a constant ? 
Or at least implement it only once here in the base class

Reply:
it is not constant, in 2 leagues it is equal to 1, but in one league it is 0.8